From 5f86cca0c8e9006213e936adf9a6ea79132a052d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@gmail.com>
Date: Mon, 16 Jun 2014 19:57:06 +0200
Subject: [PATCH] kiosk: remove invalid unref

This unref doesn't seem to be related to any reference, although it
was probably introduced in the first place to clear the floating ref,
wrongly. See following commit for a working solution.
---
 src/virt-viewer-app.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/virt-viewer-app.c b/src/virt-viewer-app.c
index 956aecb..5dc7697 100644
--- a/src/virt-viewer-app.c
+++ b/src/virt-viewer-app.c
@@ -829,7 +829,6 @@ virt_viewer_app_display_added(VirtViewerSession *session G_GNUC_UNUSED,
             if (priv->kiosk) {
                 /* don't show extra monitors that don't fit on client */
                 g_debug("kiosk mode: skip extra monitors that don't fit on client");
-                g_object_unref(display);
                 return;
             }
 
